airflow/api_fastapi/core_api/routes/public/task_instances.py

```diff
@@ -482,3 +486,91 @@ def get_mapped_task_instance_try_details(
         map_index=map_index,
         session=session,
     )
+
+
+@task_instances_router.get(
+    "/{task_id}/logs/{task_try_number}",
+    responses=create_openapi_http_exception_doc([status.HTTP_404_NOT_FOUND]),
+    response_model=None,
```

Comment:
```suggestion
```

Reply:
You can take a look at `get_config` or `get_dag_source` endpoint.

Those endpoints are also returning a plain `Response`. There is no way for FastAPI to populate properly the documentation. You can check there how this can be achieved. (appropriate `response_model` for the `application/json` response, and custom one through `responses` for the `text/plain`).

Reply:
Updated it by taking reference of the APIs - get_config/ get_dag_source, PTAL.